proxies/live/apiproxy/resources/jsc/ClientRPDetailsHeader.SetRequestHeaders.js

```diff
@@ -1,10 +1,16 @@
 var developerAppName = context.getVariable('developer.app.name');
 var developerAppId = context.getVariable('developer.app.id');
 var clientIP = context.getVariable('client.ip');
+var allowedProofingLevel = context.getVariable('developer.app.nhs-login-allowed-proofing-level');
+
+if (allowedProofingLevel === null) {
+    allowedProofingLevel = "P9";
```

Comment:
this shouldn't default to P9 ...  the backend should deal with that .. possibly it should default to null / empty string
(there are future use-cases that will be outside the P5/P9 options)